hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeDecommissionMetrics.java

```diff
@@ -97,62 +109,138 @@ public void unRegister() {
+  public void setTrackedPipelinesWaitingToCloseTotal(
           long numTrackedPipelinesWaitToClose) {
-    totalTrackedPipelinesWaitingToClose
+    trackedPipelinesWaitingToCloseTotal
             .set(numTrackedPipelinesWaitToClose);
   }
 
-  public void setTotalTrackedContainersUnderReplicated(
+  public void setTrackedContainersUnderReplicatedTotal(
           long numTrackedUnderReplicated) {
-    totalTrackedContainersUnderReplicated
+    trackedContainersUnderReplicatedTotal
             .set(numTrackedUnderReplicated);
   }
 
-  public void setTotalTrackedContainersUnhealthy(
+  public void setTrackedContainersUnhealthyTotal(
           long numTrackedUnhealthy) {
-    totalTrackedContainersUnhealthy
+    trackedContainersUnhealthyTotal
             .set(numTrackedUnhealthy);
   }
 
-  public void setTotalTrackedContainersSufficientlyReplicated(
+  public void setTrackedContainersSufficientlyReplicatedTotal(
           long numTrackedSufficientlyReplicated) {
-    totalTrackedContainersSufficientlyReplicated
+    trackedContainersSufficientlyReplicatedTotal
             .set(numTrackedSufficientlyReplicated);
   }
 
-  public long getTotalTrackedDecommissioningMaintenanceNodes() {
-    return totalTrackedDecommissioningMaintenanceNodes.value();
+  public long getTrackedDecommissioningMaintenanceNodesTotal() {
+    return trackedDecommissioningMaintenanceNodesTotal.value();
   }
 
-  public long getTotalTrackedRecommissionNodes() {
-    return totalTrackedRecommissionNodes.value();
+  public long getTrackedRecommissionNodesTotal() {
+    return trackedRecommissionNodesTotal.value();
+  }
+
+  public long getTrackedPipelinesWaitingToCloseTotal() {
+    return trackedPipelinesWaitingToCloseTotal.value();
+  }
+
+  public long getTrackedContainersUnderReplicatedTotal() {
+    return trackedContainersUnderReplicatedTotal.value();
+  }
+
+  public long getTrackedContainersUnhealthyTotal() {
+    return trackedContainersUnhealthyTotal.value();
+  }
+
+  public long getTrackedContainersSufficientlyReplicatedTotal() {
+    return trackedContainersSufficientlyReplicatedTotal.value();
+  }
+
+  public void metricRecordPipelineWaitingToCloseByHost(String host,
+                                                       long num) {
+    trackedPipelinesWaitingToCloseByHost.computeIfAbsent(host,
+            hostID -> registry.newGauge(
+                    Interns.info("trackedPipelinesWaitingToClose-" + hostID,
+                            "Number of pipelines waiting to close for "
+                                    + "host in decommissioning and "
+                                    + "maintenance mode"),
+                    0L)
+    ).set(num);
+  }
+
+  public void metricRecordOfReplicationByHost(String host,
+                               long sufficientlyReplicated,
+                               long underReplicated,
+                               long unhealthy) {
+    trackedSufficientlyReplicatedByHost.computeIfAbsent(host,
```

Comment:
Formatting is a bit strange here too. If you indent by 4 instead of 8 I think we can wrap one less line.

Reply:
Fixed - thanks again.